docker/Dockerfile

```diff
@@ -11,6 +11,21 @@ RUN wget http://github.com/begriffs/postgrest/releases/download/v${POSTGREST_VER
     mv postgrest /usr/local/bin/postgrest && \
     rm postgrest-${POSTGREST_VERSION}-ubuntu.tar.xz
 
+COPY postgrest.conf /etc/postgrest.conf
+
+
+ENV PGRST_DB_URI= \
```

Comment:
Yeah this will work pretty well as a mix of env variables and mapping files into Docker if we can just get past the stumbling block of parsing quoted integers - https://github.com/lpsmith/configurator-ng/issues/4